### PR TITLE
Create /etc/kubernetes/pki directory for vshasta

### PR DESCRIPTION
**Pull request by bklei**
#### Summary and Scope

- Fixes https://jira-pro.its.hpecorp.net:8443/browse/VSHA-402

##### Issue Type

- Bugfix Pull Request

Fix bug that was already found/fixed in metal testing (create /etc/kubernetes/pki directory).

#### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on metal (e.g. an internal system, with hardware) (x) (if yes, please include results or a description of the test)
- [ ] I tested this on vshasta (if yes, please include results or a description of the test)
 
#### Idempotency
 
N/A

#### Risks and Mitigations
 
Low -- already broken in vshasta

### CODE REVIEW

File: boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh [hunk 1]
```diff
@@ -66,6 +66,7 @@ regional = true
 multizone = true
 EOF
   echo "Copying /etc/ssl/ca-bundle.pem to /etc/kubernetes/pki/oidc.pem for kube-apiserver"
+  mkdir -p /etc/kubernetes/pki
```
Comment (rustydb) [inline boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh:69]: Nit: -pv

File: boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh [hunk 2]
```diff
@@ -66,6 +66,7 @@ regional = true
...
   cp /etc/ssl/ca-bundle.pem /etc/kubernetes/pki/oidc.pem
```
Comment (rustydb) [inline boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh:70]: Nit: -pv

Comment (rustydb) [reply boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh:70]: Will ensure umask is ignored, if applicable

File: boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh (hunk 1 above)
Comment (bklei) [reply boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh]: good catch -- fixed

**Comment by rustydb** [comment]
Rebased.

**Comment by rustydb** [comment]
Not sure how this merged..
```bash
[710]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git checkout develop
Switched to branch 'develop'
Your branch is up to date with 'origin/develop'.
[711]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git pull
git checkAlready up to date.
[712]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git checkout VSHA-402-develop
Branch 'VSHA-402-develop' set up to track remote branch 'VSHA-402-develop' from 'origin'.
Switched to a new branch 'VSHA-402-develop'
[713]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git rebase develop
First, rewinding head to replay your work on top of it...
Applying: Create /etc/kubernetes/pki directory for vshasta
Using index info to reconstruct a base tree...
M	boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh
M	boxes/ncn-node-images/kubernetes/files/scripts/metal/lib.sh
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
[714]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git status
On branch VSHA-402-develop
Your branch and 'origin/VSHA-402-develop' have diverged,
and have 6 and 1 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)

nothing to commit, working tree clean
[715]rusty@C02F43JEMD6P:~/gitstuffs/cray-shasta/node-images> git push --force-with-lease
Total 0 (delta 0), reused 0 (delta 0)
To github.com:Cray-HPE/node-images.git
 + dc1c3d184...956ae5639 VSHA-402-develop -> VSHA-402-develop (forced update)
```

I had updated it against develop and it closed.

**Comment by rustydb** [comment]
Looks like this was included in #325 here: boxes/ncn-node-images/kubernetes/files/scripts/google/lib.sh

**Pull request closed without merging**